app/src/main/res/values/strings.xml

```diff
@@ -210,11 +210,13 @@
     <string name="delete_copyright">Delete Copyright</string>
     <string name="year_validation_error">Please Enter a Valid Year</string>
     <string name="delete_confirmation_message">Are you sure you want to delete the Question?</string>
+    <string name="delete_confirmation_message_track">Are you sure you want to delete the Track?</string>
```

Comment:
Make thistemplate string?

Reply:
Sure, so should i also use the created template string for the FAQ also ?